tic-tac-toe.cmd

```diff
@@ -1,2 +1,4 @@
+REM Runs the Python app on Windows.
```

Comment:
Similarly to what I proposed in `tic-tac-toe.sh`, consider using the wording _"An application launcher for Windows Command shell and PowerShell"_. Note that I did not invent the name "Windows Command" for the built-in shell, it's how the shell is actually called in the docs: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands.

Reply:
Done.